assam_budget_board/cms_plugins.py

```diff
@@ -18,15 +18,15 @@ def render(self, context, instance, placeholder):
         tableColumns = "Head Of Account,Budget Entity,Head Description,Head Description Assamese"
         context.update({
         	"fiscalYear" : {
-        		"be" : "Budget 2019-20", 
-        		"actuals" : "Actuals 2017-18",
-				"bePrev" : "Budget 2018-19",
-				"re" : "Revised 2018-19"
+                    "be"  : "Budget %s-%s" % (settings.FISCAL_YEAR, (settings.FISCAL_YEAR+1)%1000),
+                    "actuals" : "Actuals %s-%s"% (settings.FISCAL_YEAR-2, (settings.FISCAL_YEAR-1)%1000),
+                    "bePrev" : "Budget %s-%s"% (settings.FISCAL_YEAR-1, (settings.FISCAL_YEAR)%1000),
+                    "re" : "Revised %s-%s" %  (settings.FISCAL_YEAR-1, (settings.FISCAL_YEAR)%1000)
         		},
         		'headHierarchy' : headHierarchy,
         		'tableColumns' : tableColumns,
         	})
-
+        print("Budget %s-%s" % (settings.FISCAL_YEAR, (settings.FISCAL_YEAR+1)%1000))
```

Comment:
@shreyaagrawal0809  Can we remove print from here?

Reply:
Done